map_gui/src/tools/city_picker.rs

```diff
@@ -113,15 +113,17 @@ impl<A: AppLike + 'static> CityPicker<A> {
                             "Don't see the city you want?"
                                 .draw_text(ctx)
                                 .centered_vert(),
-                            Btn::plaintext_custom(
-                                "import new city",
-                                Text::from(
-                                    Line("Import a new city into A/B Street")
-                                        .fg(Color::hex("#4CA4E5"))
-                                        .underlined(),
-                                ),
-                            )
-                            .build_def(ctx, None),
+                            ctx.style()
+                                .btn_plain_light()
+                                .label_styled_text(
+                                    Text::from(
+                                        Line("Import a new city into A/B Street")
+                                            .fg(Color::hex("#4CA4E5"))
+                                            .underlined(),
```

Comment:
A lot of the previous uses of `plaintext_custom` were simple for coloring the text, which we can achieve in the builder with `builder.label_text_color`, however we don't have first class support for `underlined`.

Though this seems to be the only place we use it, so I've opted to just use the existing `.label_styled_text`, which is sort of the spiritual successor to `plaintext_custom` (but much less used because of `label_text_color`), rather than adding somethign like `builder.underline()`. If we end up doing more underlining we can add something like that.